enable

```diff
@@ -0,0 +1,7 @@
+root_dir=`dirname $0`
```

Comment:
^ Doesn't care where it is called from, and I feel like `$root_dir` is a confusing variable name.  Maybe `$source_root` or `$srcipt_root` would be better names?